src/TR_alg.jl

```diff
@@ -92,7 +95,7 @@ function TR(
 
   xkn = similar(xk)
   s = zero(xk)
-  ψ = shifted(h, xk, Δk, χ)
+  bounded ? (ψ = shifted(h, xk, max.(-Δk,l_bound-xk), min.(Δk, u_bound-xk))) : (ψ = shifted(h, xk, Δk, χ))
```

Comment:
```suggestion
  ψ = bounded ? shifted(h, xk, max.(-Δk,l_bound-xk), min.(Δk, u_bound-xk)) : shifted(h, xk, Δk, χ)
```
but note that this line allocates. Why not store `f.meta.lvar .- xk` in `l_bound`?